.github/workflows/tests.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, windows, macos]
+    env:
+      PYTHONWARNINGS: ignore
```

Comment:
Yea, basically it was causing CI failure (instead of just logging and continuing). Happy to update this, but correcting the issue seemed pertinent for a separate PR.

Reply:
Now that I've rebased this PR, I'll go ahead and remove this to see if the dependency issues have perhaps been resolved.